packages/traceability-tests/examples/v2.json

```diff
@@ -0,0 +1,10509 @@
+      "verified": {
+        "protectedHeader": {
+          "iss": "did:jwk:eyJrdHkiOiJPS1AiLCJjcnYiOiJFZDI1NTE5IiwieCI6InpYMnpvR21XMzZUVExfa3czZy1LRlZqaDVJb2FEY29TYzA4aklENHVHcnMifQ",
+          "kid": "#0",
+          "alg": "EdDSA",
+          "cty": "vc+ld+json"
+        },
+        "payload": {
+          "@context": [
+            "https://www.w3.org/ns/credentials/v2",
+            "https://w3id.org/traceability/v1"
+          ],
+          "id": "urn:uuid:76b9ea85-6477-4160-abfd-8b5c1c53cacd",
+          "type": [
+            "VerifiableCredential",
+            "BankAccountCredential"
+          ],
+          "issuer": "did:jwk:eyJrdHkiOiJPS1AiLCJjcnYiOiJFZDI1NTE5IiwieCI6InpYMnpvR21XMzZUVExfa3czZy1LRlZqaDVJb2FEY29TYzA4aklENHVHcnMifQ",
+          "issuanceDate": "2022-04-11T16:36:24Z",
+          "credentialSubject": {
+            "type": [
+              "BankAccount"
+            ],
+            "accountId": "1111111",
+            "BIC11": "TDOMCATTTOR",
+            "familyName": "Smith",
+            "givenName": "Alice",
+            "id": "did:key:z6Mk2cd21e9abe57fae7...31073da1b522790e63834fe17a4c2be",
+            "iban": "GB74GSLD04296280001319",
+            "routingInfo": {
+              "type": [
+                "RoutingInfo"
+              ],
+              "code": "GBDSC",
+              "value": "042962"
+            },
+            "address": {
+              "type": [
+                "PostalAddress"
+              ],
+              "streetAddress": "19 Knox St",
+              "addressLocality": "Toronto",
+              "addressRegion": "ON",
+              "addressCountry": "Canada",
+              "postalCode": "M3B 1A2"
+            }
+          },
+          "credentialSchema": {
+            "id": "https://w3id.org/traceability/openapi/components/schemas/credentials/BankAccountCredential.yml",
```

Comment:
Note the self referencing credentialSchema... this is checked... AFTER ... signature verification in the tests.